qiskit/circuit/quantumcircuit.py

```diff
@@ -555,7 +555,9 @@ def draw(self, scale=0.7, filename=None, style=None, output=None,
                 gates should be placed in the output circuit if there is an option.
                 `none` results in each gate being placed in its own column. Currently
                 only supported by text drawer.
-
+            vertical_compression (string): `high`, `medium` or `low`. It merges the
+                    lines generated by `text` so the drawing will take less vertical
+                    room. Default is `medium`.
```

Comment:
We should mention that this is only for the text drawer here

Reply:
fixed in 22d37d2f